pkg/scc/scc.go

```diff
@@ -0,0 +1,384 @@
+			"name: %s, runAsUser type: %s, selinuxContext type: %s", name, runAsUser, selinuxContext)
+
+	builder := Builder{
+		apiClient: apiClient,
+		Definition: &securityV1.SecurityContextConstraints{
+			ObjectMeta: metaV1.ObjectMeta{
+				Name: name,
+			},
+			RunAsUser: securityV1.RunAsUserStrategyOptions{
+				Type: securityV1.RunAsUserStrategyType(runAsUser),
+			},
+			SELinuxContext: securityV1.SELinuxContextStrategyOptions{
+				Type: securityV1.SELinuxContextStrategyType(selinuxContext),
+			},
+		},
+	}
+
+	if name == "" {
+		glog.V(100).Infof("The name of the SecurityContextConstraints is empty")
+
+		builder.errorMsg = "SecurityContextConstraints 'name' cannot be empty"
+	}
+
+	if runAsUser == "" {
+		glog.V(100).Infof("The runAsUser of the SecurityContextConstraints is empty")
+
+		builder.errorMsg = "SecurityContextConstraints 'runAsUser' cannot be empty"
+	}
+
+	if selinuxContext == "" {
+		glog.V(100).Infof("The selinuxContext of the SecurityContextConstraints is empty")
+
+		builder.errorMsg = "SecurityContextConstraints 'selinuxContext' cannot be empty"
+	}
+
+	return &builder
+}
+
+// PullSecurityContextConstraints pulls existing SecurityContextConstraints from cluster.
+func PullSecurityContextConstraints(apiClient *clients.Settings, name string) (*Builder, error) {
+	glog.V(100).Infof("Pulling existing SecurityContextConstraints object name %s from cluster", name)
+
+	builder := Builder{
+		apiClient: apiClient,
+		Definition: &securityV1.SecurityContextConstraints{
+			ObjectMeta: metaV1.ObjectMeta{
+				Name: name,
+			},
+		},
+	}
+
+	if name == "" {
+		glog.V(100).Infof("The name of the SecurityContextConstraints is empty")
+
+		builder.errorMsg = "SecurityContextConstraints 'name' cannot be empty"
+	}
+
+	if !builder.Exists() {
+		return nil, fmt.Errorf("SecurityContextConstraints object %s doesn't exist", name)
+	}
+
+	builder.Definition = builder.Object
+
+	return &builder, nil
+}
+
+// WithPrivilegedContainer adds bool flag to the allowPrivilegedContainer of SecurityContextConstraints.
+func (builder *Builder) WithPrivilegedContainer(allowPrivileged bool) *Builder {
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		" AllowPrivilegedContainer: %t flag", builder.Definition.Name, allowPrivileged)
+
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	builder.Definition.AllowPrivilegedContainer = allowPrivileged
+
+	return builder
+}
+
+// WithPrivilegedEscalation adds bool flag to the allowPrivilegeEscalation of SecurityContextConstraints.
+func (builder *Builder) WithPrivilegedEscalation(allowPrivilegedEscalation bool) *Builder {
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		" allowPrivilegedEscalation: %t flag", builder.Definition.Name, allowPrivilegedEscalation)
+
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	builder.Definition.AllowPrivilegedContainer = allowPrivilegedEscalation
```

Comment:
This is the same as the previous method, just with a different parameter name

Reply:
+1. Fixed